### PR TITLE
Release/4.9.1

### PR DESCRIPTION
**Pull request by NachoSoto**
### Fixes:
* `CustomerInfoResponseHandler`: return `CustomerInfo` instead of error if the response was successful (#1778) via NachoSoto (@NachoSoto)
* Error logging: `logErrorIfNeeded` no longer prints message if it's the same as the error description (#1776) via NachoSoto (@NachoSoto)
* fix another broken link in docC docs (#1777) via aboedo (@aboedo)
* fix links to restorePurchase (#1775) via aboedo (@aboedo)
* fix getProducts docs broken link (#1772) via aboedo (@aboedo)

### Improvements:
* `Logger`: wrap `message` in `@autoclosure` to avoid creating when `LogLevel` is disabled (#1781) via NachoSoto (@NachoSoto)

### Other changes:
* Lint: fixed `SubscriberAttributesManager` (#1774) via NachoSoto (@NachoSoto)

### CODE REVIEW

File: Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Generated by Apple Swift version 5.6.1 (swiftlang-5.6.0.323.66 clang-1316.0.20.12)
+// Generated by Apple Swift version 5.7 (swiftlang-5.7.0.123.7 clang-1400.0.29.50)
```
Comment (NachoSoto) [inline Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h:1]: Can somebody not running macOS Ventura re-compile this with Xcode 13? 🙏🏻

Comment (taquitos) [reply Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h:1]: On it

Comment (NachoSoto) [reply Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h:1]: No rush though, we should probably wait for Monday?

Comment (taquitos) [reply Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h]: Yeah, I think we should wait to release on Monday, I'm on call so I'm here for this type of thing 😄

**Comment by NachoSoto** [comment]
Test fixes: #1783

**Review by taquitos — APPROVED** [review]
🛥🐐
download-53

**Comment by NachoSoto** [comment]
I'll rebase this when #1783 is merged and then it should be good to go on Monday.

**Pull request merged**